composer.json

```diff
@@ -127,7 +127,7 @@
         "drupal/rules": "^3.0@alpha",
         "drupal/seckit": "^1.0@alpha",
         "drupal/ui_patterns": "~1.0",
-        "drupal/view_unpublished": "^1.0@rc",
+        "drupal/view_unpublished": "dev-1.x#d3f72872dd8dd3f288dc9d63d65a740ce7d0ba9a",
```

Comment:
You like the dev version bettern than `rc1`?

Reply:
I don't really care as long as it is working. In fact, I want it to go away altogether, but I've spent way too much time on this already and did lock stanford_person to the dev version so this would keep it in parity.

Reply:
I also locked it to the version in soe_profile, and it works there fine.